src/main/java/com/notably/logic/suggestion/handler/NewSuggestionArgHandler.java

```diff
@@ -40,29 +44,36 @@ public NewSuggestionArgHandler(Model model) {
      * @return Optional.empty()
      */
     public Optional<SuggestionGenerator> handleArg(String userInput) {
+        logger.info("Starting handleArg method inside NewSuggestionArgHandler");
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(userInput, PREFIX_TITLE, PREFIX_JUMP);
 
         if (userInput.isBlank()) {
+            logger.warning("user input is blank");
```

Comment:
This shouldn't be a warning? Since it's a suggestion and it's not really an invalid thing to have a blank input